### ISSUE TITLE
Storage

### ISSUE THREAD

**Issue by JWKnapp** [xrnm/odnd-character-sheet#37]
A place to send items to from inventory when storing them someplace other than on person or animal (user feedback)

**Comment by xrnm** [comment]
Yeah, this is super important. I have an idea for a super inventory management page that is just a super condensed list of the character, all hirelings, all animals, and all strongholds and shit can just be thrown around between them.
